integration_tests/models/sql/test_unpivot_quote.sql

```diff
@@ -0,0 +1,26 @@
+{% if target.name == 'redshift' %}
+-- redshift lowercases all columns. ignoring this test for redshift
+select * from {{ ref('data_unpivot_quote_expected') }}
+
+{% else %}
```

Comment:
Does lowercasing break this test, or just make it redundant? 

I assume the former, in which case `enabled: "{{ target.name == 'redshift' }}"` is tidier on the test in your schema.yml

Reply:
Nice - this is way cleaner than the workaround I came up with